colossalai/inference/modeling/models/nopadding_llama.py

```diff
@@ -22,6 +23,14 @@
 )
 from colossalai.logging import get_dist_logger
 
+try:
+    from colossalai._C import inference_ops_cuda as inference_ops
+except:
+    inference_ops = None
+
+if inference_ops is None:
+    inference_ops = InferenceOpsLoader().load()
```

Comment:
`InferenceOpsLoader.load()` should be enough, why do you still need to check `colossalai._C`?

Reply:
Done